Community update (#314)
huggingface/blog: community-update.md

# Introducing pull requests and discussions

![Pull requests and discussions on the hub](https://res.cloudinary.com/picturesbase/image/upload/q_auto/v1653479595/Frame_30_mmccj0.jpg)

> We are thrilled to announce the release of pull requests and discussions on the Hugging Face hub!

Reviewer: ```suggestion
We are thrilled to announce the release of pull requests and discussions on the Hugging Face Hub!
```

> Pull requests and discussions are available today under the [community tab](https://huggingface.co/gpt2/discussions) and for all repositories type: models, datasets, and Spaces. Any member of the community can create and participate in discussions and pull requests.

Reviewer 2: ```suggestion
Pull requests and discussions are available today under the [community tab](https://huggingface.co/gpt2/discussions) and for all repository types: Models, Datasets, and Spaces. Any member of the community can create and participate in discussions and pull requests.
```

Author: Usually, we just uppercase Spaces because it's a product name.

> Pull requests and discussions are available today under the [community tab](https://huggingface.co/gpt2/discussions) and for all repositories type: models, datasets, and Spaces. Any member of the community can create and participate in discussions and pull requests.

Reviewer 3: ```suggestion
Pull requests and discussions are available today under the [community tab](https://huggingface.co/gpt2/discussions) and for all repository types: models, datasets, and Spaces. Any member of the community can create and participate in discussions and pull requests.
```

> It's the biggest update ever done to the HF hub and we can't wait to see the community members start collaborating with it.

Reviewer: ```suggestion
It's the biggest update ever done to the Hub, and we can't wait to see the community members start collaborating with it  🤩.
```

> We also think the new "Community" tab aligns with proposals in ethical AI stuff throughout the years. Feedback and iterations have a central place in the development of an ethical and saine machine learning software. (Those needs to exchange on datasets & models date back to the beginning of the. )We really believe having it in your toolset will unlock new kind of positive patterns in ML.

Reviewer 3: ```suggestion
We also think the new "Community" tab aligns with proposals in ethical AI stuff throughout the years. Feedback and iterations have a central place in the development of an ethical and saine machine learning software. (Those needs to exchange on datasets & models date back to the beginning of the. )We really believe having it in your toolset will unlock new kind of positive patterns in ML.
```
"ethical AI stuff" sounds a bit unprofessional to me :p

Reviewer 4: updated

> It's the biggest update ever done to the HF hub and we can't wait to see the community members start collaborating with it.
> We also think the new "Community" tab aligns with proposals in ethical AI stuff throughout the years. Feedback and iterations have a central place in the development of an ethical and saine machine learning software. (Those needs to exchange on datasets & models date back to the beginning of the. )We really believe having it in your toolset will unlock new kind of positive patterns in ML.

Reviewer 2: ```suggestion
It's the biggest update ever done to the HF hub and we can't wait to see the community members collaborating with it.
We also think the new "Community" tab aligns with ethical AI proposals throughout the years. Feedback and iterations have a central place in the development of ethical and sane machine learning software. (Those needs to exchange on datasets & models date back to the beginning of the. (the what ?) We really believe having it in your toolset will unlock new kinds of positive patterns in ML.
```

> We also think the new "Community" tab aligns with proposals in ethical AI stuff throughout the years. Feedback and iterations have a central place in the development of an ethical and saine machine learning software. (Those needs to exchange on datasets & models date back to the beginning of the. )We really believe having it in your toolset will unlock new kind of positive patterns in ML.

Reviewer 3: ```suggestion
We also think the new "Community" tab aligns with proposals in ethical AI stuff throughout the years. Feedback and iterations have a central place in the development of ethical and sane machine learning software. We really believe having it in your toolset will unlock new kinds of positive patterns in ML.
```

> We also think the new "Community" tab aligns with proposals in ethical AI stuff throughout the years. Feedback and iterations have a central place in the development of an ethical and saine machine learning software. (Those needs to exchange on datasets & models date back to the beginning of the. )We really believe having it in your toolset will unlock new kind of positive patterns in ML.

Reviewer: ```suggestion
The new "Community" tab also aligns with proposals in ethical AI throughout the years. Feedback and iterations have a central place in the development of ethical machine learning software. We really believe having it in the community's toolset will unlock new kinds of positive patterns in ML, collaborations, and progress.
```

## About discussions

![Discussions on the Hugging Face Hub](https://res.cloudinary.com/picturesbase/image/upload/v1653480977/image_9_ccvwj3.jpg)

> [Discussions](https://huggingface.co/gpt2/discussions?type=discussion) let community members ask and answer questions but also share their ideas and suggestions directly with the repository owners and the community. Anyone can create and participate in discussions in the community tab of a repository.

Reviewer: ```suggestion
[Discussions](https://huggingface.co/gpt2/discussions?type=discussion) let community members ask and answer questions as well as share their ideas and suggestions directly with the repository owners and the community. Anyone can create and participate in discussions in the community tab of a repository.
```

## About pull requests

![Pull requests on the Hugging Face Hub](https://res.cloudinary.com/picturesbase/image/upload/v1653480977/image_10_hdy9kv.jpg)

[Pull requests](https://huggingface.co/gpt2/discussions?type=pull_request) let community members open, comment, merge, or close pull requests directly from the website.

The easiest way to open a pull request is to use the "Collaborate" button in the "Files and versions" tab. It will let you do single file contributions very easily.

> 

Reviewer 2: Maybe add a screenshot here?

> Technically our Pull requests do not use forks and branches, but instead, custom "branches" called `refs` that are stored directly on the source repo. We decided to use this approach to avoid the need to create a fork for each new version of the model/dataset.

Reviewer 2: ```suggestion
Technically our Pull requests do not use forks, but instead, custom "branches" called `refs` that are stored directly on the source repo.
```

Maybe link to the relevant documentation here ?
